apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.Tests/SubscriberClientTest.cs

```diff
@@ -223,7 +223,8 @@ private class PullStream : StreamingPullStream
                     _scheduler = scheduler;
                     _writeAsyncPreDelay = writeAsyncPreDelay; // delay within the WriteAsync() method. Simulating network or server slowness.
                     var responseStream = new En(msgs, scheduler, taskHelper, clock, useMsgAsId, ct, isExactlyOnceDelivery);
-                    _call = new AsyncDuplexStreamingCall<StreamingPullRequest, StreamingPullResponse>(null, responseStream, Task.FromResult(new Metadata()), null, null, null);
+                    // Set disposeAction parameter of AsyncDuplexStreamingCall to No-op as it is called internally while disposing stream.
```

Comment:
If AsyncDuplexStreamingCall is in our control, shouldn't we instead check if the disposeAction is null and do nothing in that case?

Is there anywhere in production code that we are creating an AsyncDuplexStreamingCall passing a null disposeAction that will nos start failing as this test?

Reply:
This is in our test infrastructure. Earlier, we were not disposing the stream so `disposeAction` being null wasn't an issue. However, now that we `Dispose` the stream, this action is internally called by Grpc.Core.Api while calling and since this action is null, the `Dispose` fails.

Reply:
Thanks for the explanation